CONTRIBUTING.md

```diff
@@ -75,6 +109,55 @@ $ go test -v -short ./...
 
 ## Releasing
 
+### Notes on versioning
+
+We version the suite using the syntax `1.x.y+suite.z`, where `1.x.y` is the version of
+the Conjur server included in the suite.
+
+This works as follows:
+
+- When there is sufficient content in the Conjur OSS suite that stakeholders
+  determine a suite release is merited, a new suite release is prepared with version
+  `1.x.y+suite.z`, where `1.x.y` is the version of [Conjur](https://github.com/cyberark/conjur)
+  included in the release.
+  - Note: suite releases will not necessarily happen for every Conjur core release.
+
+- If this is the first suite release for Conjur version `1.x.y`, then the suite
+  release will be versioned `1.x.y+suite.1`, where `1.x.y` matches the included
+  Conjur core version.
+
+- At the next point in time when there is sufficient content in the Conjur OSS
+  suite that stakeholders determine a suite release is merited:
+  - If the Conjur core version in the suite is still `1.x.y`, then the next suite
+    release version will be `1.x.y+suite.2`.
+  - If the Conjur core version has changed to `1.n.m`, then the next suite
+    release version will be `1.n.m+suite.1`.
```

Comment:
```suggestion
- Subsequent suite releases using the same Conjur OSS version require that the suite version build component be incremented.  For example, a second suite release using Conjur `1.2.3` should be versioned `1.2.3+suite.2`, as the first one would have been versioned as `1.2.3+suite.1`.
```
Does read better?  I feel like it covers the cases, but correct me if I'm wrong.